editing/update-attributes/src/main/java/com/esri/samples/update_attributes/UpdateAttributesSample.java

```diff
@@ -53,11 +53,10 @@
 
 public class UpdateAttributesSample extends Application {
 
+  private ArcGISFeature selected;
```

Comment:
with your comment from my garbage collection PR in mind, should we change this to `selectedFeature`?

Reply:
Definitely! Thanks for pointing that out!